dclab/definitions/meta_const.py

```diff
@@ -124,6 +125,25 @@
         ["target duration", float, "Target measurement duration [min]"],
         ["target event count", fint, "Target event count for online gating"],
     ],
+    # All qpi-related keywords
+    "qpi": [
+        # experiment-related qpi metadata
+        ["wavelength", float, "Laser wavelength [nm]"],
+        ["medium index", float, "Refractive index of Medium"],
+        # post-analysis-related qpi metadata
+        ["software version", str, "Software version used"],
+        # calculation of pha and amp from hologram
+        ["filter_name", str, "Fourier filter used"],
+        ["filter_size", str, "Fourier filter size [1/pix]"],
+        ["filter_size_interpretation", str, "How to interpret filter size"],
+        ["sideband_freq", f2dfloatarray,
+         "Frequency coordinates of the sideband [1/pix]"],
+        # processing of hologram
```

Comment:
change "processing of hologram" to "postprocessing of phase and amplitude".
This is related to the background correction implemented in qpimage, correct? So the doc strings are also incorrect. Take a look at this: https://qpimage.readthedocs.io/en/stable/sec_code_reference.html
I think it's fine to not include `border_m` or `border_perc` . For now, I think it's also fine to not include the `mask` stuff (we would have to introduce a threshold parameter or so to make it work).
The background correction must be defined for "pha" and for "amp" separately.

Reply:
I generally use `from_mask`. The mask is a strip of pixels at the top and bottom of the mask, which indicates the background region outside of the channel.

If a uses a unique mask, we should store that mask somewhere. I have been using a feature called e.g. `mask_holo_bg`, but there is only 1 entry as the same mask is used for all holograms.

Reply:
> The background correction must be defined for "pha" and for "amp" separately.

I don't know what you mean by ^^ in practice

Reply:
> I generally use `from_mask`. The mask is a strip of pixels at the top and bottom of the mask, which indicates the background region outside of the channel.
> 
> If a uses a unique mask, we should store that mask somewhere. I have been using a feature called e.g. `mask_holo_bg`, but there is only 1 entry as the same mask is used for all holograms.

Yes, we could introduce such a mask feature. On the other hand, if it is only a strip at the top or at the bottom, maybe it would be better to just implement such a functionality in qpimage (to save processing time and disk space for .rtdc data).

Reply:
Aha thanks, makes sense.

With regard to my [comment above](https://github.com/DC-analysis/dclab/pull/227#discussion_r1192058574), what might be a good solution for dealing with reproducibility when a mask is used?

Reply:
If the mask is non-trivial (not just a strip or something that can be computed programmatically), then we would have to add a boolean mask feature for each phase and amplitude.